### PR TITLE
Update assignment and exercises for R-SQL material

### PR DESCRIPTION
**Pull request by ethanwhite**

### CODE REVIEW

**Comment by ethanwhite** [comment]
@brymz - review of this is the top priority for today. Thanks.

File: exercises/Sql-1-R.md [hunk 1]
```diff
@@ -11,13 +11,14 @@ It is clear Dr. Undómiel appreciates your skill working with large databases an
...
 1. Download a new copy of the [Portal database](http://files.figshare.com/2292171/portal_mammals.sqlite). 
-2. Connect to `portal_mammals.sqlite` using the `RSQLite` package. 
-3. Select and print out the average hind foot length and average weight of:
-  - all *Dipodomys spectabilis* individuals
-  - male *D. spectabilis*
-  - female *D. spectabilis*.
-
-
+2. Connect to `portal_mammals.sqlite` using the `RSQLite` package.
+3. Start by reminding yourself of what tables are in the database using
```
Comment (brymz) [inline exercises/Sql-1-R.md]: of what -> which ?

File: exercises/Sql-1-R.md [hunk 2]
```diff
@@ -11,13 +11,14 @@ It is clear Dr. Undómiel appreciates your skill working with large databases an
...
+`dbListTables`
```
Comment (brymz) [inline exercises/Sql-1-R.md]: I made a personal style decision to include `()` when naming functions in the exercises. In this case, '`dbListTables()`'.

File: exercises/Sql-1-R.md [hunk 3]
```diff
@@ -11,13 +11,14 @@ It is clear Dr. Undómiel appreciates your skill working with large databases an
...
+4. Then remind yourself of the fields in the `surveys` and `plots` tables using
+   `dbListFields`.
```
Comment (brymz) [inline exercises/Sql-1-R.md]: `dbListFields()`

File: exercises/Sql-2-R.md [hunk 4]
```diff
@@ -7,18 +7,16 @@ language: R
...
-frame with `species_id`, `sex`, `avg_hindfoot_length`, and `avg_weight`. 
-Your data frame should have two rows (*one row for each sex*).
-3. Automate the query using your function and a `for` loop to cycle through the 
-list of `species_id`.
-4. Store the results in a combined data frame with 
-`species_id`, `sex`, `avg_hindfoot_length`, and `avg_weight`.
\ No newline at end of file
+Produce a data frame with `species_id`, `sex`, `avg_hindfoot_length`, and
+`avg_weight` for each species.  Your data frame should have two rows for each
+species, one row for each sex.
```
Comment (brymz) [inline exercises/Sql-2-R.md]: 'R-SQL 4' was written to include `dbWriteTable()`. We could include it here or with 'R-SQL 3' if you still want it in the exercise.

**Comment by brymz** [comment]
Review complete. Excited for the test run of these new exercises.

File: exercises/Sql-2-R.md (hunk 4 above)
Comment (ethanwhite) [inline exercises/Sql-2-R.md]: R-SQL 4 is still in the assignment so let's just do it there.

**Comment by ethanwhite** [comment]
Fixes pushed. Merge when ready.

File: exercises/Sql-2-R.md (hunk 4 above)
Comment (brymz) [inline exercises/Sql-2-R.md]: Yep. I see that now. 'R-SQL 3' is dropped. Silly me.

**Pull request merged**